gradoop-examples/src/main/java/org/gradoop/examples/io/RdbmsExample.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.examples.io;
+
+import org.apache.flink.api.common.ProgramDescription;
+import org.apache.flink.api.java.ExecutionEnvironment;
+import org.gradoop.dataintegration.importer.rdbmsimporter.RdbmsDataSource;
+import org.gradoop.flink.io.impl.csv.CSVDataSink;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.gradoop.flink.util.GradoopFlinkConfig;
+
+/**
+ * Example program that converts a given relational database into a
+ * {@link LogicalGraph} and stores the resulting {@link LogicalGraph} as JSON
+ * into declared directory.
+ */
+public class RdbmsExample implements ProgramDescription {
```

Comment:
should extends `AbstractRunner` -> use `writeLogicalGraph()`

Reply:
Done